packages/launcher/src/renderer/UIComponents/OnboardContainer.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-native/no-raw-text */
```

Comment:
Could we update the ESLint config to whitelist the components we need rather than disabling this rule please?
See https://github.com/Intellicode/eslint-plugin-react-native/blob/master/docs/rules/no-raw-text.md#this-rule-has-an-object-option